lib/timescale/hyperfunctions.ex

```diff
@@ -99,4 +99,18 @@ defmodule Timescale.Hyperfunctions do
       :origin
     ])
   end
+
+  @doc """
+  Works similar to `time_bucket/2` but also activates gap-filling for the interval inferred
+  by the where clause of the query.
+
+  [Documentation](https://docs.timescale.com/api/latest/hyperfunctions/gapfilling-interpolation/time_bucket_gapfill/)
+  """
+  defmacro time_bucket_gapfill(field, time_bucket) do
```

Comment:
Would love to see `time_bucket_gapfill/2` get a `time_bucket_gapfill/3` sibling that takes options in parity with `time_bucket/3`, as I have a use case of passing timezone to my fragments 🙏

Reply:
Ah, turns out `time_bucket/3` doesn't actually support timezone. my use case calls for timezone to be passed as a positional argument.

Reply:
Okay, even with a working fragment, postgres barfs because my timestamp is a timestamp _without_ timezone, which I'd forgotten about, and which has sent me down a rabbit hole of ecto/postgres timestamp storage articles.

Disregard, though I may report back!